demo/src/app/components/+dropdown/dropdown-section.list.ts

```diff
@@ -68,6 +68,11 @@ export const demoComponentContent: ContentSection[] = [
         anchor: 'triggers-manual',
         component: require('!!raw-loader?lang=typescript!./demos/triggers-manual/triggers-manual.ts'),
         html: require('!!raw-loader?lang=markup!./demos/triggers-manual/triggers-manual.html'),
+        description: `<p>Dropdown can be triggered for two ways:
+          <ul>
+            <li>by <code>toggleDropdown</code> method</li>
```

Comment:
user may not have `toggleDropdown` method because it's only defined in our demo. You should point out that user needs to manage dropdown's `isOpen` input property

Reply:
changed